docs/other/osg-token-renewer.md

```diff
@@ -0,0 +1,195 @@
+you create as `root` with the encryption password to be used for this client
+account.
+
+For each client account, you can configure one or more `[token <TOKEN_NAME>]`
+sections, where `<TOKEN_NAME>` is a unique name of your choosing.
+These sections describe how to create the token with the `oidc-token` tool.
+For details, see
+[Creating a new OIDC Client Account](#creating-a-new-oidc-client-account)
+below.
+
+
+Details for this configuration can be found below under
+[Configuring tokens for an OIDC client account](#configuring-tokens-for-an-oidc-client-account).
+
+
+Creating a new OIDC client account
+----------------------------------
+
+The
+[Requesting Tokens](https://opensciencegrid.org/technology/software/requesting-tokens/)
+document describes the process of running `oidc-gen` as `root` to create a
+new OIDC client account.
+However, in order to keep relevant oidc-agent config files owned by the
+`osg-token-svc` service account rather than `root`, we provide a
+`osg-token-renewer-setup.sh` script.
+
+To create a new client account named `<ACCOUNT_SHORTNAME>`:
+
+- Create a corresponding file named `/etc/osg/tokens/<ACCOUNT_SHORTNAME>.pw`
+  with the encryption password to use for this client account.
+- Consult the 
+  [Requesting Tokens](https://opensciencegrid.org/technology/software/requesting-tokens/)
+  document to determine which scopes you will need for this client account.
+- Run the setup script as follows:
+  ```console
+  root@server # osg-token-renewer-setup.sh <ACCOUNT_SHORTNAME> <SCOPES...>
+  ```
+  For example,
+  ```console
+  root@server # osg-token-renewer-setup.sh myaccount123 wlcg offline_access
+  ```
+- You will be prompted on the console to visit a WLCG web link to authorize
+  the client request with a passcode printed on the console.
+  Follow the prompts (visit the web link, enter the request passcode,
+  log in with your WLCG INDIGO IAM user account, and authorize the request).
+- If this succeeds you will be prompted with a new
+  `[account <ACCOUNT_SHORTNAME>]` section to add to your `config.ini`.
+  Go ahead and add it, replacing the example section if it's still there.
+
+Next you can configure one or more tokens for this client account.
+
+
+Configuring tokens for an OIDC client account
+---------------------------------------------
+
+After you have created an OIDC client account and added it to
+`/osg/token-renewer/config.ini`, you need to create a corresponding `token`
+section in the config for each token that should be generated for this account
+(possibly with different options).
+
+Choose a `<TOKEN_NAME>` and add a new `[token <TOKEN_NAME>]` section
+(replacing the example section if it's still there).
+
+The `account` option in this section must match the `<ACCOUNT_SHORTNAME>`
+for the corresponding `[account <ACCOUNT_SHORTNAME>]` section.
+
+The `token_path` option specifies the output path where the generated token
+will be saved.
```

Comment:
So i think somewhere i mention that this is the default, and it is also spelled out like that in the example in the config.ini that gets installed with the package.  I guess the question is whether we want to let the users know that they can write tokens to other locations.  If not, i am happy with your version that just tells them what to put there.  Otherwise if they might want to write them elsewhere maybe we can present it as an option.

Reply:
I'd prefer to be prescriptive but I'm open to 

```
1.  Set the `token_path` to the output path where the generated token will be saved (recommended: `/etc/osg/tokens/<ACCOUNT_SHORTNAME>.token`) 
```